app/config/config_dev.yml

```diff
@@ -14,12 +14,20 @@ web_profiler:
 monolog:
     handlers:
         main:
-            type:   stream
-            path:   "%env(SYMFONY_LOG)%"
-            level:  info
+            type: stream
+            path: "%env(SYMFONY_LOG)%"
```

Comment:
`LOG_URL` makes sense to me. It's not only a path, it's a url used by the StreamHandler: https://github.com/Seldaek/monolog/blob/master/src/Monolog/Handler/StreamHandler.php#L50